### DISCUSSION TITLE
the fine adjustments that make a flac conversion successfull (with fre:ac)

### DISCUSSION THREAD

**Discussion by impotentfield** [enzo1982/freac#269] [Audio Format Discussion]
my experiments with flac conversions, with freac, have a lot to do with the customization part where i put my focus due to the formats special feature of apodization. any small adjustment can break or make the conversion be worth the listening quality. when i apply my apodizations with small and varying attempts of apodizing with technical series of functions due to their special effect on the quality of the end result, i either succeed or not succeed at making a correct attempt. will anyone with experience help the concept here in this discussion with flac apodization and customization ? flac more likely will end up for the user to be the prime format that will lead to quality music conversions for their leisure. any help and experience with this, for the users of fre:ac and myself, would be a great value.

**Comment by zordaz** [comment]
I remember this question was also asked at the old discussion forum. Flac is a lossless codec, adjustments of the available parameters of Flac simply can't have impact on the sound.

**Comment by impotentfield** [comment]
it seems thats not at all accurate. in my experience with programs dealing with science that are not even audio related there is a type of sorting out that people dont realize their real distinction. for instance a person can use pgp mail and not even realize that formally it was designed with a complex manner of code where inside the inner meta-boundaries of the application, there lys an intrinsic science that makes the core of the program produce the effect of the programs designation. with flac there simply isnt any realized documentation about how the apodization really produces any profound effect in the science aspect where anyway you can process in your mind, scientific intrinsic implentation applys directly from the term of apodization itself. if you cannot process the idea that an application of science is involved with apodization in flac stream conversion, then i really think you cannot because there is not real cohesive document that i myself also notice not present on xiph.org website. as i began to look through to making it up to use flac for my sony player and experiment there was no real way for having be presented information on what exactly anything was, just a horse wind of publications on simply that it was a media format and it can be contained as song media. i realize the difficulty some might have that there isnt anything producing the main idea where the science is involved with the format but as you can tell you might have gradually been informed of the horse wind information and just propped up your comment with that sort of play on misinformation. when i experiment with apodization i realized there "IS" a effectual transformation of the song streams when applying variation of functions in the conversion steps. for instance
kaiser_bessel, blackman, blackman_harris_4term_92db, bartlett_hann, hann, connes, nuttall - are specialized acoustic units
bartlett, flattop - are specialized percussion units
rectangle, triangle and hamming - are specialized extra units for mainly the idea that all the units can be used in the foreground of their scientific light to actually divise the stream to apply to another considerable scientific foreground in the manifold of the headphone technology. i honestly began experimenting and i realized the special application myself.

**Comment by zordaz** [comment]
I am afraid that you are mixing up two completely different things: lossless (audio) compression and audio processing/filtering. 

Flac was designed more than 20 years ago to be able to do bit perfect compression and decompression of audio without any loss. It is proven, well documented and used by millions on a daily basis. Flac simply can´t adjust the sound in any way, it wasn´t designed for that. You can compare it with ZIP.

Doing audio processing/filtering/apodization is perfectly fine, but it simply can't be achieved via Flac itself, you need specific software for that or develop/use specific plugins/filters for existing software like Foobar2000, Fre:ac etc.

I think https://hydrogenaud.io/ is a much more suitable forum for discussing this subject.

**Comment by impotentfield** [comment]
i have been experimenting more and it seems your statement earlier that the most fitting case for which apodization exists with flac is that it remains sound filtering as it being what it is. although thats the case it doesnt take away from my statements for which i realize empirically that the apodization does have an effect on the quality of the converted streams. my experiments are direct and i really want to maintain that it allows for a certain type of debating with peers in the arena where quality is to have fitness for main stream use for all listeners of music who want the quality to pass the test for which can maintain the industry for music on portable music players mainly. if there is no evidence of people gathering their experimentation on apodization for people who want to place for the passing of information, that makes me feel like noone is really interested in this sort of thing. i wonder why....

**Comment by zordaz** [comment]
Flac itself, and all parameters it has, simply can't alter the sound. It is technically not possible. Sound processing in any way can be done somewhere else in the playback chain of course. It might be that you are doing that with additional software or filters, that is not clear to me. If you are only experimenting with the parameters of Flac you are really wasting your time.

Again: discussions about any possible quality improvements by audio processing better belong on forums like https://hydrogenaud.io/  and the like. The people over there will be interested, but will also ask for extensive elaboration of your experiments and for well executed listing tests.
